ecosystem/indexer-grpc/indexer-grpc-table-info/src/internal_indexer_db_service.rs

```diff
@@ -155,6 +157,23 @@ impl InternalIndexerDBService {
                 Some((next_version - start_version) as i64),
                 None,
             );
+            if next_version == start_version {
```

Comment:
why moved it after the log line? this code will log once every 10ms when the db has no progress